src/styles/blocks/card.css

```diff
@@ -0,0 +1,106 @@
+.card {
+  display: flex;
+  justify-content: center;
+  gap: 30px;
+}
+
+.card__item {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: 198px;
```

Comment:
box-sizing: border-box;
and width 200px

Reply:
fixed